charts/thoras/tests/all_deployments_test.yaml

```diff
@@ -0,0 +1,27 @@
+suite: All Deployments
```

Comment:
I felt like a unit test that covers _all_ deployments would be useful, so I created this new file. I moved the things that cover many-but-not-all deployments to select_deployments file